vimeo-networking/src/main/java/com/vimeo/networking/AccountStore.java

```diff
@@ -24,25 +24,20 @@
 
 import com.vimeo.networking.model.VimeoAccount;
 
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
 /**
  * Interface responsible for handling the creation, deletion, and loading of Vimeo accounts on the client.
  * <p/>
  * Created by hanssena on 4/27/15.
  */
 public interface AccountStore {
 
+    @Nullable
     VimeoAccount loadAccount();
 
-    /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
-     */
-    @Deprecated
-    void saveAccount(VimeoAccount vimeoAccount, String email, String password);
-
-    void saveAccount(VimeoAccount vimeoAccount, String email);
+    void saveAccount(@NotNull VimeoAccount vimeoAccount, String email);
```

Comment:
Email can be null - the case for this is a client credentials "account" in which we have scope, token type, and an auth token (a VimeoAccount) but nothing that uniquely identifies the user. The email field here is the Android Account ```name``` field that will appear in the AccountManager - I can change it to accountName for clarity.